contents/spotlight/startup-langfuse.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "How Langfuse pivoted and raised $4M after leaving Y Combinator "
+date: 2024-01-16
+author: ["joe-martin"]
+showTitle: true
+rootpage: /blog
+sidebar: Blog
+hideAnchor: true
+featuredImage: ../images/blog/startup_beforesunset.png
+featuredImageType: full
+category: Startups
+---
+
+> Each month, we speak to a team enrolled in [PostHog for Startups](/startups) about the way they work. This time, we spoke to [Langfuse](https://langfuse.com/), an open source observability platform for LLMs, about their Y Combinator experience.  
+
+For the team at Langfuse, the past 12 months have been a blur of activity and highlights. After completing Y Combinator’s W23 batch they’ve earned over 1.5k [GitHub stars](/spotlight/startup-documenso), won Product of the Day on [Product Hunt](/spotlight/startup-before-sunset-ai) and raised a $4M seed round backed by Lightspeed Venture Partners. 
```

Comment:
Links to GitHub and Product Hunt still point to old articles
https://github.com/langfuse/langfuse/
https://www.producthunt.com/products/langfuse#langfuse

Reply:
```suggestion
For the team at Langfuse, the past 12 months have been a blur of activity and highlights. After completing Y Combinator’s W23 batch they’ve earned over [1.5k GitHub stars](https://github.com/langfuse/langfuse/), won Product of the Day on [Product Hunt](https://www.producthunt.com/products/langfuse#langfuse) and raised a $4M seed round backed by Lightspeed Venture Partners. 
```